fastify.js

```diff
@@ -22,6 +22,73 @@ const ContentTypeParser = require('./lib/ContentTypeParser')
 const Hooks = require('./lib/hooks')
 const loggerUtils = require('./lib/logger')
 
+const ffs = require('fast-fast-series')
```

Comment:
You can't use the exsiting `fast-iterator` package?

Reply:
Because `fast-fast-series` performs better in this case.